spec/plugins_manager_spec.coffee

```diff
@@ -0,0 +1,79 @@
+describe 'PluginsManager', ->
+  before (done) ->
+    require ['plugins_manager', 'promise'], (PluginsManager, Promise) =>
+      @plugins_manager = PluginsManager
+      @promise = Promise
+
+      @plugins_response = { plugins: [{ name: 'reviews' }] }
+
+      done()
+
+  beforeEach ->
+    @session = { shop_code: 'SA-XXXX-XXX', analytics_session: 'analytics_session' }
+    @_fetchEnabledPlugins_stub = sinon.stub(@plugins_manager::, '_fetchEnabledPlugins')
+                                      .returns((new @promise).resolve(@plugins_response))
+    @subject = new @plugins_manager(@session)
+
+    window.sa.plugins = {}
+
+  afterEach ->
+    @_fetchEnabledPlugins_stub.restore()
+    delete @session
+    delete @subject
+
+  describe '.constructor', ->
+    it 'caches the session', ->
+      expect(@subject.session).to.equal(@session)
+
+    it 'retrieves the enabled plugins', ->
+      expect(@_fetchEnabledPlugins_stub.calledOnce).to.equal(true)
+
+    it 'caches the retrieved plugins', ->
+      expect(@subject.available_plugins).to.equal(@plugins_response.plugins)
+
+  describe '#notify', ->
+    beforeEach -> @_loadScript_stub = sinon.stub(@subject, '_loadScript')
+    afterEach ->  @_loadScript_stub.restore()
+
+    context 'when action has a plugin', ->
+      context 'and plugin is enabled', ->
+        beforeEach ->
+          @_actionTriggersPlugin_stub = sinon.stub(@subject, '_actionTriggersPlugin').returns('reviews')
+          @subject.notify('addOrder', { order_id: 1 })
+
+        afterEach -> @_actionTriggersPlugin_stub.restore()
+
+        it 'make data public', ->
```

Comment:
[minor™] it 'make**s** data public', ->

Or you could be even more verbose it 'makes plugin data available in window.sa.plugins.[plugin].data'